test/run_all.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env
```

Comment:
Not intending to keep this; but added it temporarily so there's something in place to run this.